types/react-shadow-dom-retarget-events/index.d.ts

```diff
@@ -15,3 +15,5 @@ export = retargetEvents;
  * A bug is filed at [#10422](https://github.com/facebook/react/issues/10422).
  */
 declare function retargetEvents(shadowRoot: ShadowRoot): void;
+
+declare namespace retargetEvents {}
```

Comment:
I think that this change is backwards - this is an inappropriate hack.

[Please read more here from the DefinitelyTyped README](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/fbba9f55beab742b9d1c0fa9ef714e0771ad9d05#should-i-add-an-empty-namespace-to-a-package-that-doesnt-export-a-module-to-use-es6-style-imports).

But feel free to remove `esmoduleinterop` from the config and switch to `import Foo = require("foo")` style syntax